src/ssl.c

```diff
@@ -11174,6 +11171,14 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
             return ctx->error_depth;
         return SSL_FATAL_ERROR;
     }
+
+    void wolfSSL_X509_STORE_CTX_set_flags(WOLFSSL_X509_STORE_CTX* ctx,
+                                          unsigned long flags) {
```

Comment:
Function brackets should be below the function signature on their own line, as per our coding standards.  Please adjust.

If "ctx" is NULL, this function will seg fault when the pointer is drereferenced.  Please sanitize input arguments.

Also, please see comments in direct email.

Reply:
Done